### PR TITLE
chore: replace RTB <Button>, <Avatar>, <Tooltip...>

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/1331/desktop-replace-button-component
https://app.clubhouse.io/peerio/story/1330/desktop-replace-avatar-component
https://app.clubhouse.io/peerio/story/1473/desktop-create-alternative-for-tooltip-components

#### Testing instructions  
OK. Massive PR that affects almost every view in the entire app. I promise after this they get much easier. This PR replaces the RTB versions of:

* every Avatar
* every Button (not including those in Dialogs) and TooltipIconButton
* every TooltipDiv

I tested each view one-by-one as I replaced the components, but it would still be good to have eyes on them.

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/ui-library/Tooltip.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    @observable style = {
+        left: '',
+        top: ''
+    };
+
+    @action.bound setRef(ref) {
+        if (ref) {
+            this.ref = ref;
+            ref.parentElement.addEventListener('mouseenter', this.showTooltip, false);
+            ref.parentElement.addEventListener('mouseleave', this.hideTooltip, false);
```
Comment (anri-asaturov) [inline src/ui-library/Tooltip.jsx]: Unsubscribe in 'componentWillUnmount'

File: src/ui-library/Tooltip.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        }
+    }
+
+    @action.bound showTooltip() {
+        const tooltipParent = this.ref.parentElement;
```
Comment (anri-asaturov) [inline src/ui-library/Tooltip.jsx]: To be safe, check if this.ref exists

**Comment by anri-asaturov** [comment]
@ebilg this is ready to test, although... you basically need to test everything. Maybe we just merge and do it in dev (and other branches that will merge dev)?

**Comment by suprko** [comment]
^ re: Anri's last comment, I agree, I think merging and testing on dev makes things much simpler

File: src/ui-library/Tooltip.jsx [hunk 3]
```diff
@@ -42,7 +42,14 @@ class Tooltip extends React.Component {
         }
     }
 
+    componentWillUnmount() {
+        this.ref.parentElement.removeEventListener('mouseenter', this.showTooltip);
```
Comment (anri-asaturov) [inline src/ui-library/Tooltip.jsx]: check if this.ref exists.

**Pull request merged**